docs/log_collection.md

```diff
@@ -239,13 +239,15 @@ DDDatadog.verbosityLevel = DDSDKVerbosityLevelDebug;
 let logger = Logger.builder
     .sendNetworkInfo(true)
     .printLogsToConsole(true, usingFormat: .shortWith(prefix: "[iOS App] "))
+    .set(datadogReportingThreshold: .info)
     .build()
 ```
 {{% /tab %}}
 {{% tab "Objective-C" %}}
 ```objective-c
 DDLoggerBuilder *builder = [DDLogger builder];
 [builder sendNetworkInfo:YES];
+[builder setWithDatadogReportingThreshold:.info]
```

Comment:
It looks like there may be a missing semicolon at the end of this line. Could you confirm and update if needed?